storage_controller/src/main.rs

```diff
@@ -304,13 +286,9 @@ async fn async_main() -> anyhow::Result<()> {
         http_service_port: args.listen.port() as i32,
     };
 
-    // After loading secrets & config, but before starting anything else, apply database migrations
+    // Validate that we can connect to the database
     Persistence::await_connection(&secrets.database_url, args.db_connect_timeout.into()).await?;
 
-    migration_run(&secrets.database_url)
-        .await
-        .context("Running database migrations")?;
-
```

Comment:
Was this already removed from neon_local?

Reply:
Yup. `neon_local` had a stale comment

Reply:
At least `control_plane/src/storageC_controller.rs:227` still talks about running migrations (instead of just doing `createdb storage_controller` which it does).

Reply:
Good point. Done in https://github.com/neondatabase/neon/pull/8756/commits/fafd78bcb9d9a9a0ed89f32d8b64ccaaec359f62